app/controllers/admin/documentations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Admin::DocumentationsController < Admin::AdminController
+  include TranslationSaveable
+
+  def new
+    @documentation = Documentation.new(html_identifier: params[:html_identifier])
+    authorize @documentation, :new?
```

Comment:
it should already be checking `:new?` when you call the base authorization, and in the `ApplicationPolicy`, `new?` is set as an alias to `create?`, so you should be able to leave this as just `authorize @documentation` like in the `create` action.

Reply:
thanks for the catch! forgot to clear this